RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByHPSSelection.cc

```diff
@@ -258,10 +256,13 @@ PFRecoTauDiscriminationByHPSSelection::discriminate(const reco::PFTauRef& tau) c
   }
   // Now check the pizeros
   BOOST_FOREACH(const reco::RecoTauPiZero& cand, tau->signalPiZeroCandidates()) {
+    double dEta = TMath::Max(0., fabs(cand.eta() - tauP4.eta()) - cand.bendCorrEta());
+    double dPhi = TMath::Max(0., fabs(cand.phi() - tauP4.phi()) - cand.bendCorrPhi());
+    double dR = sqrt(dEta*dEta + dPhi*dPhi);
```

Comment:
don't do sqrt here, it's excessive CPU, define dR2 (and cone_size*cone_size below; multiplication is cheap)